routes/register/Register.js

```diff
@@ -0,0 +1,43 @@
+require("dotenv").config();
+
+const express = require("express");
+const router = express.Router();
+const bcrypt = require("bcryptjs");
+
+const db = require("../../data/dbConfig");
+const auth = require("../../auth/auth");
+
+function checkRegistration(req, res, next) {
+  if (
+    !req.body.username ||
+    !req.body.firstName ||
+    !req.body.lastName ||
+    !req.body.phoneNumber ||
+    !req.body.password
+  ) {
+    res
+      .status(400)
+      .json({ message: "Please enter information for all required fields." });
+  } else {
+    next();
+  }
+}
+
+function hashPassword(req, res, next) {
+  req.body.password = bcrypt.hashSync(req.body.password, 12);
```

Comment:
It's a good idea to make hash_rounds an environmental variable to keep hackers guessing ... `process.env.HASH_ROUNDS`